code/upaya/bin/setup

```diff
@@ -14,6 +14,9 @@ Dir.chdir APP_ROOT do
 
   puts "== Installing dependencies =="
   run "gem install bundler --conservative"
+
+  run 'gem install foreman && gem update foreman'
```

Comment:
maybe a noob question but why doesn't this just go in the Gemfile like the other deps?